sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -359,6 +360,7 @@ export class Application extends RecordDataModel {
     },
   )
   applicationOfferingChangeRequest?: ApplicationOfferingChangeRequest[];
+  application: { applicationPDPPDStatus: string };
```

Comment:
Why was this property added? - asking for my understanding

Reply:
I think it was added by accident :) removing.